public/components/explorer/timechart_header/timechart_header.tsx

```diff
@@ -87,9 +88,9 @@ export function TimechartHeader({
     onChangeInterval(e.target.value);
   };
 
-  if (!timeRange || !bucketInterval) {
-    return null;
-  }
+  // if (!timeRange || !bucketInterval) {
```

Comment:
Can we delete these commented-out code blocks?

Reply:
Sure, will do.